api/v1beta1/disruption_cron_webhook.go

```diff
@@ -92,8 +88,5 @@ func (d *DisruptionCron) ValidateDelete() (warnings admission.Warnings, err erro
 
 	log.Infow("validating deleted disruption cron", "spec", d.Spec)
 
-	// send informative event to disruption cron to broadcast
-	disruptionCronWebhookRecorder.Event(d, Events[EventDisruptionCronDeleted].Type, string(EventDisruptionCronDeleted), Events[EventDisruptionCronDeleted].OnDisruptionTemplateMessage)
```

Comment:
If the goal was to send the resource via an annotation, why did we need to move it to the reconcile loop? We can use the same approach in the webhook

Reply:
The webhook is okay, if the chaos-api is up and running. In the scenario where the API is down and the chaos-controller failed to sent the event and is restarted the event will never be re-emitted. By emitting from the reconciler we are sure to re-emit the events and reduce the risk of losing events.